### PR TITLE
Updating the CNI version to the lastest

### PR DESCRIPTION
**Pull request by vipul-21**
Updating the CNI version to the latest version 1.4.32

### CODE REVIEW

**Comment by vipul-21** [comment]
@rbtr

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -296,8 +296,8 @@ AMD64_ONLY_CNI_VERSIONS="
 "
 #Please add new version (>=1.4.12) in this section in order that it can be pulled by both AMD64/ARM64 vhd
 MULTI_ARCH_VNET_CNI_VERSIONS="
-1.4.22
```
Comment (chengliangli0918) [inline vhdbuilder/packer/install-dependencies.sh:299]: azure/swift-cni 1.4.22 and overlay-cni 1.4.27 is currently used in aks-rp side. I think we should cache this version in AKS VHD. 

image

**Review by chengliangli0918 — APPROVED** [review]

File: vhdbuilder/packer/install-dependencies.sh (hunk 1 above)
Comment (chengliangli0918) [inline vhdbuilder/packer/install-dependencies.sh:299]: azure cni binaries of 1.4.22 is used by aks-rp for now, we still need to cache it in VHD

**Review by chengliangli0918 — APPROVED** [review]

**Review by jaer-tsun — APPROVED** [review]
lgtm

**Pull request merged**